core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -122,7 +124,18 @@ case class SubmissionStatusResponse(
   status: SubmissionStatus,
   workflowFailureMode: Option[WorkflowFailureMode] = None
 ) {
-  def this(submission: Submission, rawlsUser: RawlsUser) = this(submission.submissionId, submission.submissionDate, rawlsUser.userEmail.value, submission.methodConfigurationNamespace, submission.methodConfigurationName, submission.submissionEntity, submission.workflows, submission.status, submission.workflowFailureMode)
+  def this(submission: Submission, rawlsUser: RawlsUser) =
+    this(
+      submissionId = submission.submissionId,
+      submissionDate = submission.submissionDate,
+      submitter = rawlsUser.userEmail.value,
+      methodConfigurationNamespace = submission.methodConfigurationNamespace,
+      methodConfigurationName = submission.methodConfigurationName,
+      submissionEntity = submission.submissionEntity,
+      workflows = submission.workflows,
+      status = submission.status,
+      workflowFailureMode = submission.workflowFailureMode
+    )
```

Comment:
@davidangb will the orchestration swagger need to be updated as well?

Reply:
yup, orch swagger will also need updates!

Reply:
Here's the issue for tracking: 
https://app.zenhub.com/workspace/o/databiosphere/firecloud-app/issues/63